packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -63,7 +63,7 @@ export class GasEstimator {
   private lastFastPriceGwei = 0;
   private latestMaxFeePerGasGwei: number;
   private latestMaxPriorityFeePerGasGwei: number;
-  private latestBaseFee: number;
```

Comment:
more consistent to store the base fee as a Gwei number, like the other variables in this class.